src/mSupplyMobileApp.js

```diff
@@ -153,6 +153,8 @@ class MSupplyMobileAppContainer extends React.Component {
   onAuthentication = user => {
     const { dispatch } = this.props;
     dispatch(UserActions.login(user));
+    // Required to refresh menuApp after language is updated (which is only on login page)
+    dispatch(UserActions.setLanguage(UIDatabase.getSetting(SETTINGS_KEYS.CURRENT_LANGUAGE)));
```

Comment:
Nice, this works great.

I *think* it makes more sense for me to `connect` `LoginModal` component to get `dispatch`, and then `dispatch` the action in `onSelectLanguage`?

Reply:
Ok - I'll have a look on doing this change now.

Reply:
Hmmm seems like a slight big change because `LoginModal` is a class and not a react component, which I don't think works well with `connect`. I'll have to look into that with a little more time - TBD tomorrow.

Reply:
Done! Finally with Chris' help. The class `LoginModal` was being exported at the top and as default, but the top one was the one in use. Sorted now, merging!